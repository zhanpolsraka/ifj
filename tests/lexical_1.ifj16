//1
//Chybnost' uvazovek - lexicka chyba 1
class Main{
	static String str = "Hello, FIT
	;
	static void run(){

	}
}