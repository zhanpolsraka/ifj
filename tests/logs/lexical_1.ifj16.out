
Lexical error! Unknown lexem ""Hello, FIT
	;
	static void run(){

	}
}ÿ"!

